src/main/kotlin/no/skatteetaten/aurora/boober/service/DeployService.kt

```diff
@@ -7,10 +7,7 @@ import no.skatteetaten.aurora.boober.service.openshift.OpenShiftClient
 import no.skatteetaten.aurora.boober.service.openshift.OpenShiftResponse
 import no.skatteetaten.aurora.boober.service.resourceprovisioning.ExternalResourceProvisioner
 import no.skatteetaten.aurora.boober.service.resourceprovisioning.ProvisioningResult
-import no.skatteetaten.aurora.boober.utils.addIfNotNull
-import no.skatteetaten.aurora.boober.utils.deploymentConfigFromJson
-import no.skatteetaten.aurora.boober.utils.imageStreamFromJson
-import no.skatteetaten.aurora.boober.utils.openshiftKind
+import no.skatteetaten.aurora.boober.utils.*
```

Comment:
Are you using the wrong codeStyle here?

Reply:
I do use the Aurora-CodeStyle, perhaps wrong version? I will fix this